internal/cmd/login.go

```diff
@@ -225,13 +226,44 @@ func updateInfraConfig(client *api.Client, loginReq *api.LoginRequest, loginRes
 }
 
 func oidcflow(host string, clientId string) (string, error) {
+	// find out what the authorization endpoint is
```

Comment:
We can use this library to automatically find the authorization endpoint that the CLI should open in the browser for the provider. It uses the `.well-known/oidc` endpoint to do this.